state_file.py

```diff
@@ -12,7 +12,7 @@ class StateFile:
 
     def __init__(self, cluster_name: str, path: str) -> None:
         self.cluster_name = cluster_name
-        self.path = path
+        self.path = path.normpath(path) + "/" + cluster_name
```

Comment:
We have 1 state file for all clusters now. Having a separate path for each cluster makes it hard to track.